src/commons/sagas/BackendSaga.ts

```diff
@@ -758,7 +747,7 @@ const newBackendSagaTwo = combineSagaHandlers(SessionActions, {
 
 function* oldBackendSagaOne(): SagaIterator {
   yield takeEvery(
-    DELETE_SOURCECAST_ENTRY,
+    deleteSourcecastEntryAction,
```

Comment:
```suggestion
    deleteSourcecastEntryAction.type,
```

Reply:
Thanks for catching this! Resolved in 6adfe5b4691242ca1d54e328c4cdab656f07b22c.